kostyor/tests/unit/rest_api/test.py

```diff
@@ -339,3 +340,61 @@ def test_discover_cluster_discovery_failed_error_response(self,
                                               auth_url='http://9.9.9.9')
         fake_session.assert_called_once_with(auth=password_mock)
         fake_discover.assert_called_once_with()
+
+    @mock.patch.object(db_api, 'get_cluster', mock.Mock())
+    @mock.patch.object(db_api, 'get_hosts_by_cluster', mock.Mock())
+    def test_host_list_cluster_exists_success(self):
+        fake_hosts = ['hostname_1', 'hostname_2']
+        db_api.get_hosts_by_cluster.return_value = fake_hosts
+
+        res = self.app.get('/clusters/1234/hosts')
+
+        self.assertEqual(200, res.status_code)
+        hosts = json.loads(res.data.decode('utf-8'))
+        self.assertEqual(fake_hosts, hosts)
+        db_api.get_cluster.assert_called_once_with('1234')
+        db_api.get_hosts_by_cluster.assert_called_once_with('1234')
+
+    @mock.patch.object(db_api, 'get_cluster', mock.Mock())
+    @mock.patch.object(db_api, 'get_hosts_by_cluster', mock.Mock())
+    def test_host_list_wrong_cluster_id_404(self):
+        db_api.get_cluster.side_effect = Exception("Wrong cluster ID.")
+
+        res = self.app.get('/clusters/fake/hosts')
+
+        self.assertEqual(404, res.status_code)
+        db_api.get_cluster.assert_called_once_with('fake')
+        self.assertEqual(False, db_api.get_hosts_by_cluster.called)
```

Comment:
minor: Here and below, I think it's better to use a special assert function: `assertFalse`.